server/templates/email_article_body.html

```diff
@@ -0,0 +1,21 @@
+<html>
+<body>
+<h1>{% if 'place' in article and article.place %}{{ article.place[0].name }}:&nbsp;{% endif %}{{ article.headline | safe }}</h1>
+Published At : {{ article.versioncreated | format_datetime(date_format='%c') }}
+{% if article.associations is defined and article.associations.featuremedia is defined %}
+<br>
+    <img src="CID:MainImage">
```

Comment:
You can place a style here to allow the image to scale based on the available width of the browser/screen:
`style="height: auto;width: 100%;max-width: max-content;"`

This will keep the aspect ratio and not scale bigger than the original size of the image.